src/runtime/manifest-parser.pegjs

```diff
@@ -678,9 +698,16 @@ NameWithColon
     return name;
   }
 
+PaxelMode
+  = '=' {
+    pushPaxelMode(true);
+    return text();
+  }
+
 ParticleHandleConnectionBody
-  = name:NameWithColon? direction:(Direction '?'?)? whiteSpace type:ParticleHandleConnectionType annotations:SpaceAnnotationRefList? maybeTags:SpaceTagList? expression:(whiteSpace? '=' multiLineSpace PaxelExpression)?
+  = name:NameWithColon? direction:(Direction '?'?)? whiteSpace type:ParticleHandleConnectionType annotations:SpaceAnnotationRefList? maybeTags:SpaceTagList? expression:(whiteSpace? PaxelMode multiLineSpace PaxelExpression)?
   {
+    popPaxelMode();
```

Comment:
should you only pop() if expression was actually specified? What if expression was not defined for the connection?

Reply:
good catch, yes.